pkg/tools/opentelemetry/exporter.go

```diff
@@ -52,8 +54,8 @@ func InitSpanExporter(ctx context.Context, exporterURL string) trace.SpanExporte
 	return exporter
 }
 
-// InitMetricExporter - returns an instance of OpenTelemetry Metric Exporter.
-func InitMetricExporter(ctx context.Context, exporterURL string) *sdkmetric.Exporter {
+// InitOPTLMetricExporter - returns an instance of OpenTelemetry Metric Exporter.
+func InitOPTLMetricExporter(ctx context.Context, exporterURL string) sdkmetric.Reader {
```

Comment:
I'm not sure about the change because it breaks compatibility with cmd apps.

Could you explain how is critical to change signature?

Reply:
The open telemetry SDK returns a metric.Reader for Prometheus for the prometheus exporter. 
https://github.com/open-telemetry/opentelemetry-go/blob/v1.16.0/exporters/prometheus/exporter.go#L53 
From what I understood with the open telemetry SDK, `metric.Reader` is more generic way to be used as way of collecting data than `sdkmetric.Exporter`

Reply:
OK, got it.
Do you need the change in release v1.11.0?

Reply:
@szvincze ?

Reply:
Right now I don't think it is critical to have it in 1.11.0.

Reply:
@LionelJouin , @szvincze OK, let's target this for release 1.12.0 because this change will require updating all cmd apps due to the changing contract of function InitMetricExporter -> InitOPTLMetricExporter

Reply:
> @LionelJouin , @szvincze OK, let's target this for release 1.12.0 because this change will require updating all cmd apps due to the changing contract of function InitMetricExporter -> InitOPTLMetricExporter

@denis-tingaikin: We had some internal discussions about this topic. It seems this change would save quite significant effort for us if we could get it in 1.11.0. Is it possible to consider it in 1.11.0? Is there other alternative to get it before 1.12.0?

Reply:
Sure, but we'll need a help with updating cmd-repos.

Also, we supposed to be ready for delaying in release if something went wrong.

@edwarnicke Don't you mind if we consider this PR in this release (v.1.11.0)?